ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Block/Abstract.hs

```diff
@@ -108,6 +109,10 @@ data family CodecConfig blk :: Type
 -- avoid circular dependencies.
 data family StorageConfig blk :: Type
 
+-- | Credentials needed for block forging. In eras that use KES, this will be
+-- a pair of KES sign key and OpCert; in other eras, it should be 'Void'.
+type family BlockForgingCredentials blk :: Type
```

Comment:
This is defined here, and for the HFBlock, but nowhere else?

Reply:
I think this is a leftover from a previous design that we don't actually need anymore. Will delete.